ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/system-control-plane.j2

```diff
@@ -0,0 +1,44 @@
+{% if system.control_plane is arista.avd.defined %}
+
+## System Control-Plane
+
+{%    if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
```

Comment:
indentation in this file is not following our guidelines. Please use +4 spaces per level.
Example:
```jinja2
{% if level1 %}
{%     if level2 %} <- 5 spaces in total
```

Reply:
Fixed indentation for `system-control-plane.j2`